README.adoc

```diff
@@ -101,7 +96,7 @@ of them allow to communicate via Unix domain sockets. Unix domain sockets
 usually are just files, so standard Unix file permissions apply to them.
 
 IP sockets also have the disadvantage that other programs on the same host are
-able to connect to them, unless you use complicated netfilter rules or network
+unable to connect to them, unless you use complicated netfilter rules or network
```

Comment:
Maybe the wording should be somewhat different to be less confusing, but "able" is actually right, because what you usually don't want here, is that users or applications that shouldn't have access to the socket are *able* to connect to it (which is usually the case for IP sockets).